performant-software/Annotation-Studio
# Internal route

## What does this PR do?
1. Users who are not logged in are notified to login rather than being greeted with a "This page doesn't exist" message.

## How to test?
1. Go to http://staging.covecollective.org/ if you are not logged in otherwise log out
2. You should not see a "This page doesn't exist" message.
